data/truffle_chocolate_stout.json

```diff
@@ -0,0 +1,139 @@
+{
+  "name": "B-Sides - Truffle and Chocolate Stout",
```

Comment:
I think the convention is to mark it as a B-Side after the title, see the three that I committed the other day.